label_studio/data_manager/actions/predictions_to_annotations.py

```diff
@@ -42,6 +46,7 @@ def predictions_to_annotations(project, queryset, **kwargs):
     logger.debug(f'{count} predictions will be converter to annotations')
     db_annotations = [Annotation(**annotation) for annotation in annotations]
     db_annotations = Annotation.objects.bulk_create(db_annotations)
+    Task.objects.filter(id__in=tasks_ids).update(updated_at=now())
```

Comment:
Maybe it's better to move into `TaskSerializerBulk.post_process_annotations` ?

Reply:
It will add a database update query, that we don't need on import. If it's ok, I can move.

Reply:
Got it.

Reply:
Why do we need this line if we already have set_updated_at = True for `perform_action`?

You can put task_ids in `return {'response_code': 200, 'detail': f'Created {count} annotations',  ==> 'task_ids': xxx}`

and then use them [here ](https://github.com/heartexlabs/label-studio/pull/1784/files#diff-8ff0ffe1df554b71bae2a1554d8ce537efd879e691ea46211339893d561a1232R113)

don't forget to do `result.pop('task_ids')`